jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/PathHistoryPointBuilder.java

```diff
@@ -44,7 +44,7 @@ public static J2735PathHistoryPoint genericPathHistoryPoint(JsonNode pathHistory
       
       // Required elements
       if (latOffset == -131072) {
-         php.setLatOffset(null);
+         php.setLatOffset(BigDecimal.valueOf(-131072));
```

Comment:
With swapping the null for an actual value here, will that create unexpected values to come through? If we left it as null and tweaked our JSON ToString method to ignore nulls it would simply not serialize it, but here we're including this new negative value.

Reply:
The idea behind this is we need to know when RSUs and OBUs are generating data that is incorrect. If we see them fail to generate required J2735 fields, we need to know this because there is something wrong with the device itself. It is perfectly legal for them to use an unavailable value but if we are converting it to null we lose that visibility.